package.json

```diff
@@ -0,0 +1,141 @@
+{
+  "name": "climate-change-lab",
+  "version": "0.0.1",
+  "description": "A climate change modelling data and graphical visualizations lab by Azavea",
+  "keywords": [
+    "climate change lab",
+    "climate change",
+    "lab",
+    "Azavea"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "tslint": "tslint",
+    "webpack": "webpack",
+    "webpack-dev-server": "webpack-dev-server",
+    "protractor": "protractor",
+
+    "build": "npm run build:dev",
+      "prebuild:dev": "npm run clean:dist",
+    "build:dev": "webpack --config config/webpack.dev.js --progress --profile --colors --display-error-details --display-cached",
+      "prebuild:prod": "npm run clean:dist",
+    "build:prod": "webpack --config config/webpack.prod.js  --progress --profile --colors --display-error-details --display-cached --bail",
+
+    "server": "npm run server:dev",
+    "server:dev": "webpack-dev-server --config config/webpack.dev.js --progress --profile --colors --watch --display-error-details --display-cached --content-base src/ --host 0.0.0.0 --port 3100",
+    "server:dev:hmr": "npm run server:dev -- --inline --hot",
+
+    "lint": "npm run tslint \"src/**/*.ts\"",
+
+    "test": "karma start",
+
+    "docs": "npm run typedoc -- --options typedoc.json --exclude '**/*.spec.ts' ./src/",
+
+    "start": "npm run server:dev",
+    "start:hmr": "npm run server:dev:hmr"
+   },
+
+  "dependencies": {
+    "@angular/http": "2.0.0-rc.4",
+    "@angular/common": "2.0.0-rc.4",
+    "@angular/compiler": "2.0.0-rc.4",
+    "@angular/core": "2.0.0-rc.4",
+    "@angular/forms": "0.2.0",
+    "@angular/platform-browser": "2.0.0-rc.4",
+    "@angular/platform-browser-dynamic": "2.0.0-rc.4",
+    "@angular/platform-server": "2.0.0-rc.4",
+
+    "ie-shim": "^0.1.0",
+    "core-js": "^2.4.0",
+    "rxjs": "5.0.0-beta.6",
+    "zone.js": "~0.6.12",
+
+    "http-server": "^0.9.0",
+
+    "jquery": "2.2.2",
```

Comment:
I added in some typical packages, which in the past includes jquery. Right now the dropdowns are done via jquery but obviously that won't be the case. I'll clean these up though, I don't think we need all the dependencies listed.